tasks/docker-compose.yml

```diff
@@ -17,7 +17,8 @@
     state: absent
 
 - name: Docker login (must "--no-include-email")
-  shell: "$(aws ecr get-login --region {{ aws_default_region }} --no-include-email)"
+  shell: "$(aws ecr get-login-password | docker login --username AWS \
```

Comment:
Why didn't use `command`?

Reply:
?

Reply:
if possible we still use shell code at this point